### PR TITLE
Added field enable_volume_profiling for cluster object

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/gluster-integration#353
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

File: tendrl/commons/objects/definition/master.yaml [hunk 1]
```diff
@@ -252,6 +252,9 @@ namespace.tendrl:
         node_identifier:
           help: How nodes should be identified while create cluster
           type: list
+        enable_volume_profiling:
+          help: Whether to enable volume profiling
+          type: Boolean
```
Comment (r0h4n) [inline tendrl/commons/objects/definition/master.yaml]: Please add "is_managed" as a boolean

File: tendrl/commons/objects/cluster/__init__.py [hunk 2]
```diff
@@ -5,7 +5,8 @@ class Cluster(objects.BaseObject):
     def __init__(self, integration_id=None, public_network=None,
                  cluster_network=None, node_configuration=None,
                  conf_overrides=None, node_identifier=None, sync_status=None,
-                 last_sync=None, is_managed="no", *args, **kwargs):
+                 last_sync=None, is_managed="no",
```
Comment (r0h4n) [inline tendrl/commons/objects/cluster/__init__.py]: use bool for "is_managed"

File: tendrl/commons/objects/definition/master.yaml (hunk 1 above)
Comment (shtripat) [inline tendrl/commons/objects/definition/master.yaml]: Done. Will add that.

File: tendrl/commons/objects/cluster/__init__.py (hunk 2 above)
Comment (shtripat) [inline tendrl/commons/objects/cluster/__init__.py]: Done.

**Pull request merged**